File: examples/python/otp_verify.py
import os
import sys

from kalourmade_arkesel import ArkeselClient

if len(sys.argv) < 3:
    sys.exit("Usage: python otp_verify.py <number> <code>")
number, code = sys.argv[1], sys.argv[2]
client = ArkeselClient(os.environ["ARKESEL_API_KEY"])

result = client.otp.verify(code=code, number=number)

print(f"Verified: {result.message}")

File: examples/node/otpVerify.ts
import { ArkeselClient } from '@kalourmade/arkesel-sms';

const [number, code] = process.argv.slice(2);
if (!number || !code) {
  console.error('Usage: tsx otpVerify.ts <number> <code>');
  process.exit(1);
}

const client = new ArkeselClient(process.env.ARKESEL_API_KEY!);
const result = await client.otp.verify({ code, number });

console.log(`Verified: ${result.message}`);
